src/tools/pbs_python.c

```diff
@@ -2382,36 +2382,36 @@ main(int argc, char *argv[], char *envp[])
 		int ll;
 		char *pc, *pc2;
 
-#ifdef SYSTEM_PYTHON_PATH
-		pbs_asprintf(&python_path, "%s", SYSTEM_PYTHON_PATH);
-		pc = strdup(SYSTEM_PYTHON_PATH);
-		if (pc == NULL) {
-			fprintf(stderr, "Out of memory\n");
-			free(python_path);
-			return 1;
-		}
-		pc2 = strstr(pc,"bin/python");
-		if (pc2 == NULL) {
-			fprintf(stderr, "Python executable not found!\n");
-			free(python_path);
-			return 1;
-		}
-		*pc2 = '\0';
-		if (strlen(pc) > 0) {
-			snprintf(python_prefix, MAXPATHLEN, "%s", pc);
-			free(pc);
-		} else {
-			fprintf(stderr, "Python home not found!\n");
-			free(python_path);
-			return 1;
-		}
-		snprintf(python_envbuf, MAXBUF, "%s=%s", PYHOME, python_prefix);
-#else
 		snprintf(python_prefix, sizeof(python_prefix), "%s/python",
 			pbs_conf.pbs_exec_path);
-		pbs_asprintf(&python_path, "%s/bin/python", python_prefix);
-		snprintf(python_envbuf, MAXBUF, "%s=%s", PYHOME, python_prefix);
-#endif
+		pbs_asprintf(&python_path, "%s/bin/python3", python_prefix);
```

Comment:
Here's another place where we hardcode python3. I think the goal here should be to make the transition to python4 as easy as possible. Our future selves will thank us.